src/model/beneficial.owner.gov.model.ts

```diff
@@ -22,6 +22,7 @@ export const BeneficialOwnerGovKeys: string[] = [
 
 export interface BeneficialOwnerGov {
   id: string;
+  link?: Links,
```

Comment:
I asked for Links to be removed from MO mappings for now as not in BOs. Let's add this separately.

Reply:
Links removed